vizro-core/docs/pages/user-guides/selectors.md

```diff
@@ -26,20 +26,106 @@ For more details, kindly refer to the documentation of the underlying dash compo
-    boolean values cannot be displayed properly as labels in the underlying dash components.
+    boolean values cannot be displayed properly as labels in the underlying Dash components.
 
 ## Numerical selectors
 
-For more details, kindly refer to the documentation of the underlying dash components:
+For more details, refer to the documentation of the underlying Dash components:
 
-- [dcc.Slider](https://dash.plotly.com/dash-core-components/slider])
-- [dcc.RangeSlider](https://dash.plotly.com/dash-core-components/rangeslider])
+- [`dcc.Slider`](https://dash.plotly.com/dash-core-components/slider])
+- [`dcc.RangeSlider`](https://dash.plotly.com/dash-core-components/rangeslider])
 
-???+ note
+!!! note
 
     When configuring the [`Slider`][vizro.models.Slider] and the [`RangeSlider`][vizro.models.RangeSlider] with float values, and using `step` with an integer value, you may notice
     unexpected behavior, such as the drag value being outside its indicated marks.
     To our knowledge, this is a current bug in the underlying [`dcc.Slider`](https://dash.plotly.com/dash-core-components/slider) and
     [`dcc.RangeSlider`](https://dash.plotly.com/dash-core-components/rangeslider) component, which you can circumvent by adapting the `step` size accordingly.
 
-To enhance existing selectors, please see our How-to-guide on creating [custom components](custom-components.md)
+To utilize `numerical` [`Filter`][vizro.models.Filter] selectors, the filtered column must be of `numeric` dtype,
+indicating that [pandas.api.types.is_numeric_dtype()](https://pandas.pydata.org/docs/reference/api/pandas.api.types.is_numeric_dtype.html) must return `True` for the filtered column.
+
+## Temporal Selectors
+
+For more details, refer to the documentation of the underlying Dash components:
+
+- [`dmc.DateRangePicker`](https://www.dash-mantine-components.com/components/datepicker#daterangepicker)
+- [`dmc.DatePicker`](https://www.dash-mantine-components.com/components/datepicker)
+
+!!! note
+
+    [`DatePicker`][vizro.models.DatePicker] component is based on two underlying components `dmc.DatePicker` and `dmc.DateRangePicker`.
+    When [`DatePicker`][vizro.models.DatePicker] is configured with `range=True` (the default), the underlying component is `dmc.DateRangePicker`; when `range=False` the component is `dmc.DatePicker`.
+    When configuring the [`DatePicker`][vizro.models.DatePicker] make sure to provide your dates for `min`, `max` and `value` arguments in `"yyyy-mm-dd"` format or as datetime type.
+
+To utilize `temporal` [`Filter`][vizro.models.Filter] selectors, the filtered column must be of `datetime` dtype,
+indicating that [pandas.api.types.is_datetime64_any_dtype()](https://pandas.pydata.org/docs/reference/api/pandas.api.types.is_datetime64_any_dtype.html) must return `True` for the filtered column.
+
+!!! note
+
+    Dataframe column type can be changed to datetime using:
```

Comment:
```suggestion
    `pd.DataFrame` column types can be changed to `datetime` using:
```
but please check this again, currently it renders ugly

Reply:
maybe even skip this, or make this a one liner with the first option only for simplification